code-challenges.js

```diff
@@ -39,28 +55,60 @@ var myNumbers2 = [8, -7, 0, 6, 2]
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
+const noVowels = (string) => {
+  let newArray = string.split("")
+  let filteredArray = newArray.filter(value => {
+    return value !== "a" && value !== "e" && value !== "i" && value !== "o" && value !== "u"
+  })
+  let newString = filteredArray.join("")
+  return newString
+}
+console.log(noVowels(stringWithVowels2))
```

Comment:
Almost there!!! This would output `ILvJvScrpt`, however if we look at the EXPECTED output is, we should be receiving `LvJvScrpt`.

This means, its catching a capital vowel, the i specifically. This can be fixed if we look into our filter and the conditions we're checking. I'd look into .toLowerCase() and applying it to each value check.